example/android/app/src/release/java/com.example/ReactNativeFlipper.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * <p>This source code is licensed under the MIT license found in the LICENSE file in the root
+ * directory of this source tree.
+ */
+package example;
```

Comment:
Is it valid package and file's directory? Shouldn't it be java/com/example?

Reply:
@exzos28 yes you’re right, it should be com.example. Will push the fix.

Reply:
Not that we expect someone to release this lib on play store 😅